src/async/deserializeAsync.ts

```diff
@@ -42,13 +42,22 @@ export function createTsonParseAsyncInner(opts: TsonAsyncOptions) {
 		}
 	}
 
-	return async (iterable: AsyncIterable<string>) => {
+	return async (stream: ReadableStream<Uint8Array>) => {
```

Comment:
maybe
```ts
	return async (stream: ReadableStream<Uint8Array>) => {
    stream: 	ReadableStream<Uint8Array> | AsyncIterable<string> | ReadableStream<Uint8Array>
```

Reply:
- we don't care that much about bundle size for this to matter
- it'd be nicer dx?
- they can be converted from/to each other seamlessly AFAIK